float32 learningRate
int32 indexCategory
---
